test/views/ViewSpecBase.scala

```diff
@@ -75,13 +85,19 @@ trait ViewSpecBase extends SpecBase {
   def assertContainsText(doc: Document, text: String): Assertion =
     assert(doc.toString.contains(text), "\n\ntext " + text + " was not rendered on the page.\n")
 
+  def assertNotContainsText(doc: Document, text: String): Assertion =
+    assert(!doc.toString.contains(text), "\n\ntext " + text + " was not rendered on the page.\n")
+
   def assertElementHasText(element: Element, text: String): Assertion =
     assert(element.text.contains(text), "\n\ntext " + text + " was not rendered in the element.\n")
 
-  def assertContainsMessages(doc: Document, expectedMessageKeys: String*): Unit =
-    for (key <- expectedMessageKeys) assertContainsText(doc, messages(applicationBuilder().build())(key))
+  def assertContainsMessages(doc: Document, expectedMessageKeys: String*): Unit      =
+    for (key <- expectedMessageKeys) assertContainsText(doc, messages(app)(key))
```

Comment:
Feel app isn't needed here. Can't this just be `for (key <- expectedMessageKeys) assertContainsText(doc, messages(key))`?